microshift_troubleshooting/microshift-troubleshoot-updates.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="microshift-troubleshoot-updates"]
+= Troubleshoot {product-title} updates
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-troubleshoot-updates
+
+toc::[]
+
+To troubleshoot {product-title} updates, use the following guide.
+
+[IMPORTANT]
+====
+You can only update {product-title} from one minor version to the next in sequence. For example, you must update 4.14 to 4.15.
+====
+
+include::modules/microshift-updates-troubleshooting.adoc[leveloffset=+1]
+
+//additional resources for troubleshooting module
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Because the `microshift-updates-troubleshooting` module has multiple subsections, it currently looks like this "Additional resources" list applies specifically to the "Manual RPM update failed", the way that it renders.

Suggest changing this to the `== Additional resources` [style](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies) (in this case, `===`, I think) so that it can look like the last subsection of the "Troubleshooting MicroShift updates" section (and therefore apply to all of the subsections), if that is the intention:

```suggestion
[role="_additional-resources"]
[id="additional-resources_microshift-updates-troubleshooting"]
=== Additional resources
```